src/views/header/ConservationView.js

```diff
@@ -2,10 +2,14 @@ const view = require("backbone-viewj");
 const dom = require("dom-helper");
 import * as svg from "../../utils/svg";
 
+// this allows lots of flexibility wrt specifying colors
+// but adds ~25K to msa.min.gz ...
+var d3_scale; // = require "d3-scale" ;
```

Comment:
The problem is that if we define `d3_scale` here, we will overwrite the global variable (which the distributed package provides